wca-guidelines.md

```diff
@@ -156,7 +156,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-A><speedsolving><speedsolving> Article A: Speed Solving
 
 - A1a2+) [ADDITION] In case of a cumulative time limit, the judge records the original recorded time for a DNF on the score sheet in parentheses, e.g. "DNF (1:02.27)".
-- A1a2++) [EXAMPLE] If a round has a cumulative time limit of 30 minutes and the competitor's results first two attempts are 6:00 and DNF (10:00), then the time limit for the third attempt is 30:00 - 6:00 - 10:00 = 14:00. For this particular example, note that only the final result for an attempt counts. Therefore, a third attempt with an original recorded time of 13:59 without penalties meets the time limit because the final result is 13:59 (see [Regulation 9f2](regulations:regulation:9f2)), while a third attempt with a result of 13:59 + 2 = 14:01 does not meet the time limit.
+- A1a2++) [EXAMPLE] If a round has a cumulative time limit of 30 minutes and the competitor's first two results are 6:00 and DNF (10:00), then the time limit for the third attempt is 30:00 - 6:00 - 10:00 = 14:00. For this particular example, note that only the final result for an attempt counts. Therefore, a third attempt with an original recorded time of 13:59 without penalties meets the time limit because the final result is 13:59 (see [Regulation 9f2](regulations:regulation:9f2)), while a third attempt with a result of 13:59 + 2 = 14:01 does not meet the time limit.
```

Comment:
This should've been a separate PR but Github doesn't allow me to create branches (I think?). I changed "competitor's results first two attempts" to "competitor's first two results"

Reply:
Hi, sorry about this late reply.  We want to keep the same terminology in the Regulations and the Guidelines.  So in this case, the "attempt" should be kept.  But this sentence do needs modification.  How about "....and the competitor's results **of the** first two attempts..."
Could you make the change?